content/docs/reference/design-docs/stdcm/signaling_interface/index.fr.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Interface avec la signalisation"
+linkTitle: "3 - Interface avec la signalisation"
+weight: 30
+---
+
+Cette interface est encore en cours de conception.
```

Comment:
why is this file not translated ? Is it because it is still a wip ? Maybe a `index.en.md` with just `WIP` inside could do the job.